iphone/Classes/TiUIiPadPopoverProxy.m

```diff
@@ -388,7 +342,7 @@ - (void)updateContentViewWithSafeAreaInsets:(NSValue *)insetsValue
         UIEdgeInsets edgeInsets = [insetsValue UIEdgeInsetsValue];
         viewController.view.frame = CGRectMake(viewController.view.frame.origin.x + edgeInsets.left, viewController.view.frame.origin.y + edgeInsets.top, viewController.view.frame.size.width - edgeInsets.left - edgeInsets.right, viewController.view.frame.size.height - edgeInsets.top - edgeInsets.bottom);
       },
-      NO);
+      YES);
```

Comment:
This will cause the main thread to block, is this intended?

Reply:
Yes, This will fix layout issue when you open popover from barButtonItem